core/src/main/scala/cromwell/core/ExecutionStatus.scala

```diff
@@ -2,21 +2,24 @@ package cromwell.core
 
 object ExecutionStatus extends Enumeration {
   type ExecutionStatus = Value
-  val NotStarted, QueuedInCromwell, Starting, Running, Failed, RetryableFailure, Done, Bypassed, Aborted = Value
-  val TerminalStatuses = Set(Failed, Done, Aborted, Bypassed)
+  val NotStarted, QueuedInCromwell, Starting, Running, Aborting, Failed, RetryableFailure, Done, Bypassed, Aborted, UnReachable = Value
+  val TerminalStatuses = Set(Failed, Done, Aborted, Bypassed, UnReachable)
   val TerminalOrRetryableStatuses = TerminalStatuses + RetryableFailure
+  val NonTerminalStatuses = values.diff(TerminalOrRetryableStatuses)
 
   implicit val ExecutionStatusOrdering = Ordering.by { status: ExecutionStatus =>
     status match {
       case NotStarted => 0
       case QueuedInCromwell => 1
       case Starting => 2
       case Running => 3
-      case Aborted => 4
-      case Bypassed => 5
-      case RetryableFailure => 6
-      case Failed => 7
-      case Done => 8
+      case Aborting => 4
+      case UnReachable => 5
```

Comment:
Is unreachable 2 words? :)

Reply:
Argh I knew I forgot to fix this somewhere :p
Also congratulations you just officially started the seagulling ! Let it rain ! : smile:

Reply:
It's what i do

Reply:
That's not rain.

Reply:
🐦 🐦 🐦 
; ; ; ; ;
  ☔️
  😨

Reply:
(could it be `Unstartable` or `InterruptedBeforeStarting` or `AintGonnaHappen`?)